<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1059</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout" rowstretch="1,1,10,1,1" columnstretch="1,10,5,1">
    <item row="1" column="3">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>30</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="0" rowspan="2">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>32</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="2">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>17</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="2" rowspan="3">
     <widget class="QTableWidget" name="tableWidget"/>
    </item>
    <item row="4" column="2">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>17</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="1" rowspan="2">
     <widget class="QLabel" name="label">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QTableWidget" name="tableWidget_2"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1059</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionopen_2"/>
    <addaction name="actionsave"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>查看</string>
    </property>
    <widget class="QMenu" name="menu_Histogram">
     <property name="title">
      <string>Histogram
</string>
     </property>
     <addaction name="actionR"/>
     <addaction name="actionG_channel"/>
     <addaction name="actionB_channel"/>
     <addaction name="actionGray"/>
    </widget>
    <addaction name="menu_Histogram"/>
    <addaction name="actionpattle"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>编辑</string>
    </property>
    <widget class="QMenu" name="menureduction">
     <property name="title">
      <string>reduction</string>
     </property>
     <addaction name="actionto_8bits_colorful"/>
     <addaction name="actionto_8bits_gray"/>
    </widget>
    <addaction name="actionrevoke"/>
    <addaction name="actionredo"/>
    <addaction name="separator"/>
    <addaction name="action_Brightness"/>
    <addaction name="actioninverse"/>
    <addaction name="menureduction"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionopen">
   <property name="text">
    <string>open...</string>
   </property>
  </action>
  <action name="actionsave">
   <property name="text">
    <string>save...</string>
   </property>
  </action>
  <action name="actionsave_as">
   <property name="text">
    <string>save as...</string>
   </property>
  </action>
  <action name="actionR">
   <property name="text">
    <string>R channel</string>
   </property>
  </action>
  <action name="actionG_channel">
   <property name="text">
    <string>G channel</string>
   </property>
  </action>
  <action name="actionB_channel">
   <property name="text">
    <string>B channel</string>
   </property>
  </action>
  <action name="actionGray">
   <property name="text">
    <string>Gray</string>
   </property>
  </action>
  <action name="action_Brightness">
   <property name="text">
    <string>Brightness...</string>
   </property>
  </action>
  <action name="actionInfomation">
   <property name="text">
    <string>Infomation</string>
   </property>
  </action>
  <action name="actioninverse">
   <property name="text">
    <string>inverse</string>
   </property>
  </action>
  <action name="actionpattle">
   <property name="text">
    <string>pattle</string>
   </property>
  </action>
  <action name="actionto_8bits_colorful">
   <property name="text">
    <string>to 8bits color</string>
   </property>
  </action>
  <action name="actionto_8bits_gray">
   <property name="text">
    <string>to 8bits gray</string>
   </property>
  </action>
  <action name="actionrevoke">
   <property name="text">
    <string>undo</string>
   </property>
  </action>
  <action name="actionredo">
   <property name="text">
    <string>redo</string>
   </property>
  </action>
  <action name="actionopen_2">
   <property name="text">
    <string>open...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
